src/content/developers/docs/web2-vs-web3/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Web2 vs Web3
+description:
+lang: en
+sidebar: true
+---
+
+Web2 refers to the version of the internet most of us know today. An internet dominated by companies that provide services in exchange for your personal data. Web3, in the context of Ethereum, refer to decentralized apps that run on the blockchain. These are apps that allow anyone to participate without monetising their personal data.
+
+## Web3 benefits
+
+Many Web3 developers have chosen to build dapps because of Ethereum's inherent decentralization:
+
+- Anyone who is on the network has permission to use the service – or in other words, permission isn't required.
+- No one can block you or deny you access to the service.
+- Payments are built in via the native token, Ether (ETH).
+- Ethereum is turing-complete, meaning you can pretty much program anything.
+
+## Practical comparisons
+
+| Web2                                                                       | Web3                                                                                                           |
+| -------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| Twitter can censor any account or tweet                                    | Web3 tweets would be uncensorable because control is decentralized                                             |
```

Comment:
@samajammin IIUC, then this means that an _adversarial_ entity would need to force web3 node owners to turn off their servers in order to force censorship, yes?

I mean, if we run with the Twitter analogy, then today, Twitter Inc can be legally challenged by governments to forcefully take down info/tweets about protests, grassroots organization, etc but in a web3 world, a Twitter DApp can't be placed under the same constraints because the "Twitter app" & "Twitter service" are wholly owned by the EVM system.  Therefore the only way to _take down_ tweets/info is by forcefully turning off node providers. Right?

I'm asking to make sure I understand this correctly.